lms/static/scripts/frontend_apps/components/test/ContentSelector-test.js

```diff
@@ -63,18 +76,17 @@ describe('ContentSelector', () => {
     const noop = () => {};
     return mount(
       <Config.Provider value={fakeConfig}>
-        <ContentSelector
-          setErrorInfo={noop}
-          onSelectContent={noop}
-          {...props}
-        />
+        <ContentSelector onError={noop} onSelectContent={noop} {...props} />
       </Config.Provider>
     );
   };
 
   const isLoadingIndicatorVisible = wrapper =>
     wrapper.exists('FullScreenSpinner');
 
+  const onNextExecutionCycle = () =>
```

Comment:
We have a function called `delay` which is used in quite a few tests in the client which is basically the same as this, but with a configurable delay. `delay(0)` is quite a common idiom to wait one macro-task.

The term "execution cycle" is a not completely clear. Something like "event loop term" would be more specific, as it differentiates this from something like `Promise.resolve()` which waits only for the next microtask execution, whereas this waits for the next macrotask execution.